sensing/autoware_pointcloud_preprocessor/include/autoware/pointcloud_preprocessor/distortion_corrector/distortion_corrector_node.hpp

```diff
@@ -52,12 +52,23 @@ class DistortionCorrectorComponent : public rclcpp::Node
   std::string base_frame_;
   bool use_imu_;
   bool use_3d_distortion_correction_;
+  bool update_azimuth_and_distance_;
+
+  std::optional<AngleConversion> angle_conversion_opt_;
+
+  /// @brief Counts the number of pointclouds for which angle conversion failed.
+  int angle_conversion_failure_num_{0};
+
+  /// @brief The maximum number of failed angle conversions due to pointclouds not containing enough
+  /// points before throwing an error.
+  static constexpr int failure_tolerance{20};
```

Comment:
I left a comment below about this point

Reply:
removed in https://github.com/autowarefoundation/autoware.universe/pull/8380/commits/4bc0df1ea17e80b390bca04ba2f66fd0df39afdb